docs/vector.rst

```diff
@@ -0,0 +1,141 @@
+.. _vector:
+
+Vector
+======
+
+
+``ti.Vector`` is actually a kind of ``ti.var``, but have multiple compoment variables within it.
```

Comment:
I think your next paragraph has explained the idea, so maybe this is duplicative?